lib/clearsale/order_response.rb

```diff
@@ -15,23 +15,17 @@ class OrderResponse
     attr_reader :order_id, :status, :score
 
     def self.build_from_send_order(package)
-      new(package.fetch(:package_status, {}))
+      new(package.fetch('Orders', {}).first)
     end
 
     def self.build_from_update(package)
```

Comment:
Why do you have two different methods that do the same exact thing? Remove this and update the usage.